### PR TITLE
 Support optional disable flag for overviews

### PR DESCRIPTION
**Pull request by tylerparsons**

### CODE REVIEW

File: scripts-available/CDB_Overviews.sql [hunk 1]
```diff
@@ -35,15 +35,28 @@ AS $$
...
+    SELECT COALESCE(value::text = 'true', false)
+    FROM cdb_conf WHERE key = 'disable_overviews'
+    INTO disable_overviews;
+
+    IF disable_overviews THEN
+      RETURN QUERY SELECT
+        NULL::regclass AS base_table,
+        NULL::integer AS z,
+        NULL::regclass AS overview_table
+      WHERE false;
```
Comment (gfiorav) [inline scripts-available/CDB_Overviews.sql:49]: Interesting! Why do you use `WHERE false;` here? What's the result of omitting `WHERE false` altogether?

**Review by gfiorav — COMMENTED** [review]
I posted a question on the use of `WHERE false`. Great tests, solid addition 👍

File: scripts-available/CDB_Overviews.sql (hunk 1 above)
Comment (tylerparsons) [reply scripts-available/CDB_Overviews.sql:49]: `WHERE false;` ensures that 0 rows are returned, which is the desired result.  Omitting this clause would give 1 row with `NULL` values.

Comment (gfiorav) [reply scripts-available/CDB_Overviews.sql:49]: very interesting! Thanks the explanation

**Review by gfiorav — APPROVED** [review]

File: scripts-available/CDB_Overviews.sql (hunk 1 above)
Comment (vmanohar1) [reply scripts-available/CDB_Overviews.sql:49]: Makes perfect sense.. 
select  NULL::regclass AS base_table,NULL::integer AS z,NULL::regclass AS overview_table where false;
 base_table | z | overview_table 
------------+---+----------------
(0 rows)
select  NULL::regclass AS base_table,NULL::integer AS z,NULL::regclass AS overview_table ;
 base_table | z | overview_table 
------------+---+----------------
            |   | 
(1 row)

**Comment by vmanohar1** [comment]
Where are you planning of adding -

**Comment by vmanohar1** [comment]
Where are you planning of adding -
INSERT INTO cdb_conf VALUES ('disable_overviews', 'true'::json);

**Comment by vmanohar1** [comment]
It is certainly needed in library DB..
It will be good to add that to Sample and Org DB as well..

**Comment by tylerparsons** [comment]
That step will be done manually. The intent is not to change the default behavior of the extension.

**Comment by vmanohar1** [comment]
Rest of the things look good..

**Pull request merged**